.storybook/manager-head.html

```diff
@@ -1 +1,24 @@
+<script id="google-consent-mode" data-cookieconsent="ignore">
```

Comment:
I think the google tag script should come first, followed by the cookiebot script and then everything else
```suggestion
<script async src="https://www.googletagmanager.com/gtag/js?id=G-FK58C1XN5P"></script>
<script
    id="cookiebot"
    src="https://consent.cookiebot.com/uc.js"
    data-cbid='7cfd3819-2ae8-4de9-bf89-6ed28fb8bce4'
    data-blockingmode="auto"
    type="text/javascript"
    defer></script>
```